src/Uno.UI/Extensions/PermissionsHelper.Android.cs

```diff
@@ -19,11 +19,185 @@ public static void Initialize()
 					.CreateAsync<string, bool>(TryGetPermissionCore)
 					.LockInvocation(InvocationLockingMode.Share);
 
-			Windows.Extensions.PermissionsHelper.Initialize(getPermission, CheckPermission);
-		}
+			Windows.Extensions.PermissionsHelper.Initialize(getPermission, CheckPermission, MissingPermissions, TryAskPermissionAsync);
+		}
+
+		private static int _permissionRequest;
+		#region "submethods for MissingPermissions"

```

Comment:
There is no need for regions

Reply:
but it can help in code understanding...